exwm-input.el

```diff
@@ -102,6 +102,13 @@ defined in `exwm-mode-map' here."
 (defconst exwm-input--update-focus-interval 0.01
   "Time interval (in seconds) for accumulating input focus update requests.")
 
+(defconst exwm-input--passthrough-functions '(read-char
```

Comment:
I've defined this as a private constant instead of as a `defcustom` as this list _should_ be exhaustive and I figured we could make it public later.